src/ralph/domains/filters.py

```diff
@@ -0,0 +1,16 @@
+# # -*- coding: utf-8 -*-
```

Comment:
when https://github.com/allegro/ralph/pull/1807 will be merged, these filters won't be necessary